app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -62,12 +57,28 @@ public function showTenantRegistrationForm()
     {
         return view('auth.register', [
             'user_type' => Role::TENANT,
-            'faculties' => Faculty::all(),
-            'workshops' => Workshop::all(),
             'countries' => require base_path('countries.php'),
         ]);
     }
 
+    public const USER_RULES = [
```

Comment:
Mainly restructure only.
The Educational Information are not handled by the Registration Controller from now on.

Reply:
Yes. But we can trick this: register them as guest and then modify them.

Reply:
I do not want to merge this without the application-handling/viewer possibilities.